UnitySDK/Assets/ML-Agents/Scripts/Policy/BarracudaPolicy.cs

```diff
@@ -32,7 +32,7 @@ public class BarracudaPolicy : IPolicy
             NNModel model,
             InferenceDevice inferenceDevice)
         {
-            var aca = GameObject.FindObjectOfType<Academy>();
+            var aca = Object.FindObjectOfType<Academy>();
```

Comment:
Why this change ?

Reply:
The static method is actually on Object, not GameObject